src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -412,11 +413,12 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 					if (isInteractiveProgressTreeData(part)) {
 						partsToRender[index] = part;
 					} else {
-						const wordCountResult = this.getDataForProgressiveRender(element, part);
+						const wordCountResult = this.getDataForProgressiveRender(element, part, { renderedWordCount: 0, isFullyRendered: false, lastRenderTime: 0, isPlaceholder: isPlaceholderMarkdown(part) });
```

Comment:
This seems like a lot, and maybe is a side-effect of `isPlaceholder` not belonging on that interface, since I don't think that the method should depend on that property